earth_enterprise/src/server/wsgi/serve/publish/publish_manager.py

```diff
@@ -154,9 +155,8 @@ def HandleResetRequest(self, request, response):
       response: response object.
     """
     logger.debug("HandleResetRequest...")
-    request_url = request.GetParameter(constants.ORIGIN_REQUEST_HOST)
-    assert request_url
```

Comment:
We could probably keep the assert commands, just use it for geserver_url instead of request_url

Reply:
ok, Will do that.